### PR TITLE
Fixes Scrolling related issues in Search (#39,#11) and uniformises Navbar

### PR DESCRIPTION
**Pull request by smkvr98**

### CODE REVIEW

File: DPRHTML/index.css [hunk 1]
```diff
@@ -79,7 +79,7 @@ body {
 }
 
 hr {
-  width: 100vw;
+  /*width: 100vw;*/
```
Comment (parthopdas) [inline DPRHTML/index.css]: Remove comments.

File: DPRHTML/search-results.html [hunk 2]
```diff
@@ -66,7 +66,7 @@
       </div>
     </div>
 
-    <div class="row">
+    <div class="row"><div id="paliTextContent">
```
Comment (parthopdas) [inline DPRHTML/search-results.html]: Format please.

Also please ensure that 'para' parameter works. It has a dependency on paliTextContext
Refer my previous commit.

File: DPRHTML/search.html [hunk 3]
```diff
@@ -44,13 +44,14 @@
         }
       });
     }
+    searchHandler = (event)=>{
```
Comment (parthopdas) [inline DPRHTML/search.html]: Nice. Parenthesis around event is superfluous

File: DPRHTML/styles.css [hunk 4]
```diff
@@ -236,16 +236,15 @@ left:20px;
 
 
 #paliTextContent.COLLAPSE{
-	overflow-y: scroll;
-	height:calc(99vh - 4Rem - 250px); /*screenheight - approx height of topbar , bottombar (decided by font 1.5Rem each and bottomFrameContent (255px)*/
-  transition: height 0.3s;
+	height:calc(100vh - 6Rem - 250px);
```
Comment (parthopdas) [inline DPRHTML/styles.css]: Format the tabs vs spaces. Let's use spaces.

**Comment by smkvr98** [comment]
deals with #39 and #11

**Pull request merged**